.github/workflows/checks.yml

```diff
@@ -203,6 +203,7 @@ jobs:
 
       - name: Run licensei
         env:
-           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         run: |
+          export GITHUB_TOKEN
```

Comment:
do we still need that `export`?

Reply:
It is was fixes the CI

Reply:
Marking the PR draft again to re-verify this...